File: 7kyu/(7 kyu) Reverse words/(7 kyu) Reverse words.go
package kata

import "strings"

func ReverseWords(str string) string {
  words := strings.Split(str, " ")
  res := ""
  for _, w := range words {
    r := ""
    for _, s := range w {
        r = string(s) + r
    }
    res = res + " " + r
  }
  return res[1:]
}

File: 7kyu/(7 kyu) Reverse words/(7 kyu) Reverse words.py
# #1
# def reverse_words(text):
#     words = text.split(" ")
#     result = []
#     for w in words:
#         result.append(w[::-1])
#     return " ".join(result)

# #2


def reverse_words(text):
    return " ".join(text[::-1].split(" ")[::-1])
